### PR TITLE
completed ticket 8

### PR DESCRIPTION
**Pull request by Amir2341**

### CODE REVIEW

File: __tests__/app.test.js [hunk 1]
```diff
@@ -191,3 +191,28 @@ describe("/api/users", () => {
...
+        expect(articles).toHaveLength(12);
+        expect(articles).toEqual(
+          expect.arrayContaining([
+            expect.objectContaining({
+              article_id: expect.any(Number),
+              comment_count: expect.any(Number),
+              title: expect.any(String),
+              topic: expect.any(String),
+              author: expect.any(String),
+              body: expect.any(String),
```
Comment (Poonam-raj) [inline __tests__/app.test.js]: no `body` key is required in this response, if you think about it from a user POV, getting the body from every article is gonna be a massive amount of information to see at once - by omitting it we make our lives easier as users (if they want to see the body they can select by article id)

File: __tests__/app.test.js [hunk 2]
```diff
@@ -191,3 +191,28 @@ describe("/api/users", () => {
...
+              created_at: expect.any(String),
+              votes: expect.any(Number),
+            }),
+          ])
+        );
```
Comment (Poonam-raj) [inline __tests__/app.test.js:215]: `the articles should be sorted by date in descending order.`
^ from trello

I'd like to see some testing here to check if the default order is what we expect it to be

**Review by Poonam-raj — CHANGES_REQUESTED** [review]
Some comments around testing to consider but otherwise looking good

I'd like you to send another PR once you've addressed them (mainly so the default sorting can be looked over) -thanks!

**Comment by cjplatten** [comment]
looks good!

**Pull request merged**